Gemfile.lock

```diff
@@ -315,76 +316,78 @@ GEM
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
+    crack (0.4.6)
+      bigdecimal
+      rexml
     crass (1.0.6)
-    dalli (3.2.3)
+    dalli (3.2.7)
+      base64
     ddtrace (0.45.0)
       msgpack
     debug_inspector (0.0.3)
     diffy (3.4.2)
-    dogstatsd-ruby (3.0.0)
-    domain_name (0.5.20190701)
-      unf (>= 0.0.5, < 1.0.0)
-    doorkeeper (5.6.6)
+    dogstatsd-ruby (3.3.0)
```

Comment:
### [dogstatsd-ruby](https://github.com/DataDog/dogstatsd-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.0.0 → 3.3.0

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/dogstatsd-ruby/3.0.0/3.3.0)_]





<details>
<summary>Commits</summary>

A change of **59** commits. See the full changes on [the compare page](https://github.com/DataDog/dogstatsd-ruby/compare/v3.0.0...v3.3.0).

These are the first 10 commits:
- (31e5494) [Adds failing spec for symbol tag value](https://github.com/DataDog/dogstatsd-ruby/commit/31e54941dd240ccf6e9353879835eec422a097ae)
- (3bc17a5) [`to_s` all the tags](https://github.com/DataDog/dogstatsd-ruby/commit/3bc17a54ee53b5ad53d4ae677f1586cbd37161e5)
- (7f8d8b7) [Ignore nil tags and convert symbol ones](https://github.com/DataDog/dogstatsd-ruby/commit/7f8d8b740f071766956a8c844e183a1e787e45c3)
- (0bf5236) [Add unix socket support](https://github.com/DataDog/dogstatsd-ruby/commit/0bf5236ea448a934b24d71f49946f5d1863eb35b)
- (c64b371) [Use nonblocking send for unix socket](https://github.com/DataDog/dogstatsd-ruby/commit/c64b371f8e8cd6bbcb8362c2e113ae51ef91888d)
- (1ee2a5f) [Use defaults when host/port are nil](https://github.com/DataDog/dogstatsd-ruby/commit/1ee2a5f4fd3b02824b521253d9be081868170e0f)
- (8b816f8) [Don't send anything when the buffer is empty](https://github.com/DataDog/dogstatsd-ruby/commit/8b816f89748cd1365fe542610b2a7153822e612d)
- (bdf6065) [Merge pull request #53 from pschambacher/pschambacher/backwa…](https://github.com/DataDog/dogstatsd-ruby/commit/bdf606578e24da642548ffd6f3a93f471a5274e6)
- (30d0e0e) [Merge pull request #51 from jacobbednarz/to-s-all-the-tags](https://github.com/DataDog/dogstatsd-ruby/commit/30d0e0e5932715084d0cc87bf5de335335472592)
- (518fa60) [Fix error handling when using Unix Socket](https://github.com/DataDog/dogstatsd-ruby/commit/518fa60b437ff561a54cc0b3f96f26d0b1029467)


</details>